base/multi.jl

```diff
@@ -1429,17 +1429,18 @@ let nextidx = 0
     global chooseproc
     function chooseproc(thunk::Function)
         p = -1
-        env = thunk.env
-        if isa(env,Tuple)
-            for v in env
-                if isa(v,Box)
-                    v = v.contents
-                end
-                if isa(v,AbstractRemoteRef)
-                    p = v.where; break
-                end
-            end
-        end
+        # TODO jb/functions
```

Comment:
This was kind of a hack: if we found a RemoteRef inside a closure, we used its location as a hint as to where the spawn should run. I'm not totally sure this is useful in practice.